vowpalwabbit/parse_regressor.cc

```diff
@@ -349,8 +351,20 @@ void save_load_header(
         else  // < VERSION_FILE_WITH_INTERACTIONS
         {
           // pairs and triples may be restored but not reflected in interactions
-          all.interactions.insert(std::end(all.interactions), std::begin(all.pairs), std::end(all.pairs));
-          all.interactions.insert(std::end(all.interactions), std::begin(all.triples), std::end(all.triples));
+          std::vector<std::vector<uint8_t>> newpairs_t;
```

Comment:
Why is the variable name suffixed with _t?

Reply:
Will change that